public/locales/en/common.json

```diff
@@ -147,7 +147,7 @@
   "tinRequired": "Tax Identification Number is Required",
   "tinText": "TIN:",
   "returnTo": "Return to",
-  "donateWithPlanetCash": "Donate with PlanetCash",
+  "donateWithPlanetCash": "Pay with PlanetCash",
```

Comment:
This should also be changed in LingoHub - otherwise I will be changed back again after the next export.